### ISSUE TITLE
Train with new loss function

### ISSUE THREAD

**Issue by zghn** [CMU-Perceptual-Computing-Lab/openpose_train#42]
Hi,

I’m new in deep learning implementation. Theoretically I’ve defined a new loss function to train the openpose model more precise. I trained the model which had implemented in keras for about 20 epochs. The results decreased 1.5 percent. what’s wrong with it? I should train more? or my loss function is not good?

Data: COCO 2014
Training setting:    multisgd, base_lr=2e-5, batch_size=10, train_sample = 117576, val_samples = 2476
In addition, I finetuned model with existing weights which result AP = 68.9% .

So thanks,
